Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -677,8 +677,12 @@ def SwapKeepAndDrop(l):
                               ])
 phase2_tracker.toModify(FEVTDEBUGHLTEventContent,
                         outputCommands = FEVTDEBUGHLTEventContent.outputCommands+[
+                            'keep *_hltSiPixelClusters_*_*',
+                            'keep *_hltSiPhase2Clusters_*_*',
                             'keep *_hltPhase2PixelTracks_*_*',
-                            'keep *_hltPhase2PixelVertices_*_*'
+                            'keep *_hltPhase2PixelVertices_*_*',
+                            'keep *_hltGeneralTracks_*_*',
```

Comment:
What is the size increasing with this hltGeneralTracks collection? FEVTDEBUGHLT is super big already, and this will effect the relvals size and Phase-2 production.

Reply:
> What is the size increasing with this hltGeneralTracks collection? 

Zero. This is just a renaming.